tests/integration/security/filebased_tls_origination/destination_rule_tls_test.go

```diff
@@ -127,7 +128,7 @@ spec:
 				BuildOrFail(t)
 
 			for _, tt := range []string{"grpc", "http", "tcp"} {
-				ctx.NewSubTest(tt).Run(func(ctx framework.TestContext) {
+				t.Run(tt, func(t *testing.T) {
```

Comment:
@nmittler what is the implication of this change? I feel like either we do need it, and should revert this change, or we don't need it and should remove `ctx.NewSubTest`?

Reply:
It doesn't like it when I try adding test features flag to these NewSubTests either